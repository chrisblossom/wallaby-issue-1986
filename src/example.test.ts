'use strict';

import fs from 'fs';
import path from 'path';

test('javascript test file exists', () => {
  expect({ first: 1 }).toMatchSnapshot();
  expect({ second: 2 }).toMatchSnapshot();

  const snapshotFileTSPath = path.resolve(__dirname, '__snapshots__/example.test.ts.snap');
  const snapshotFileTSExists = fs.existsSync(snapshotFileTSPath);
  expect(snapshotFileTSExists).toEqual(true);

  const snapshotFileJSPath = path.resolve(__dirname, '__snapshots__/example.test.js.snap');
  const snapshotFileJSExists = fs.existsSync(snapshotFileJSPath);
  expect(snapshotFileJSExists).toEqual(false);
});

