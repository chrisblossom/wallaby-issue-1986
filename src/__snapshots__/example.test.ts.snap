// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`javascript test file exists 1`] = `
Object {
  "first": 1,
}
`;

exports[`javascript test file exists 2`] = `
Object {
  "second": 2,
}
`;
